include/zephyr/bluetooth/bluetooth.h

```diff
@@ -538,6 +589,17 @@ size_t bt_data_get_len(const struct bt_data data[], size_t data_count);
  */
 size_t bt_data_serialize(const struct bt_data *input, uint8_t *output);
 
+/**
+ * @brief Local Bluetooth LE controller features and capabilities.
+ *
+ * @details This struct provides details about the Bluetooth LE controller's supported features,
+ * states, and various other capabilities. It includes information on ACL and ISO data packet
+ * lengths, the controller's resolving list size, and the maximum advertising data length. This
+ * information can be obtained after enabling the Bluetooth stack with @ref bt_enable().
```

Comment:
```suggestion
 * information can be obtained after enabling the Bluetooth stack with @ref bt_enable().
```

Ditto, either ref or ()